src/game/QuizBuilder.ts

```diff
@@ -31,10 +38,21 @@ export class QuizBuilder {
      */
     poseQuestions(typeOfAnswers: Question[]): Question[] {
         const picks: Question[] = [];
+        
+        //loads correct answered questions
+        const nonRepeatQuestions = localStorage.getItem('nonRepeatQuestions');
+        let notThisQuestionAgain: string[] = [];
+        if(nonRepeatQuestions !== null){
+            notThisQuestionAgain = JSON.parse(nonRepeatQuestions);
+        }
+
         for (let index = 0; index < 2; index++) {
             let pickNumber: number = this.randomNumber(typeOfAnswers);
 
-            while (picks.includes(typeOfAnswers[pickNumber])) {
+            //when question appears twice or has already been answered correctly
```

Comment:
This code snippet would make a good function which can then be tested isolated. What do you think?

Reply:
it sounds too simple